Plots/Contours/NCL_conOncon_5.py

```diff
@@ -0,0 +1,138 @@
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.path as mpath
+import matplotlib.pyplot as plt
+import matplotlib.ticker as mticker
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and
+# load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/HGT500_MON_1958-1997.nc"),
+                     decode_times=False)
+
+###############################################################################
+# Plot:
+
+# Generate a figure
+fig = plt.figure(figsize=(8, 8))
+
+# Create an axis with a polar stereographic projection
+ax = plt.axes(projection=ccrs.NorthPolarStereo())
+
+# Add land feature to map
+ax.add_feature(cfeature.LAND, facecolor='lightgray')
+
+# Set extent to include latitudes between 0 and 40 and longitudes between
+# -180 and 180 only
+ax.set_extent([-180, 180, 0, 40], ccrs.PlateCarree())
+
+# Set draw_labels to False so that you can manually manipulate it later
+gl = ax.gridlines(
+    ccrs.PlateCarree(),
+    draw_labels=False,
+    linestyle="--",
+    linewidth=1,
+    color='darkgray',
+    zorder=2)
+
+# Manipulate latitude and longitude gridline numbers and spacing
+gl.ylocator = mticker.FixedLocator(np.arange(0, 90, 15))
+gl.xlocator = mticker.FixedLocator(np.arange(-180, 180, 30))
+
+# Set boundary to a circle
+theta = np.linspace(0, 2 * np.pi, 100)
+center, radius = [0.5, 0.5], 0.5
+verts = np.vstack([np.sin(theta), np.cos(theta)]).T
+circle = mpath.Path(verts * radius + center)
+ax.set_boundary(circle, transform=ax.transAxes)
+
+# Manipulate longitude labels (0, 30 E, 60 E, ..., 30 W, etc.)
+ticks = np.arange(0, 210, 30)
+etick = ['0'] + [r'%dE' %
+                 tick for tick in ticks if (tick != 0) & (tick != 180)] + ['180']
+wtick = [
+    r'%dW' %
+    tick for tick in ticks if (
+        tick != 0) & (
+            tick != 180)]
+labels = etick + wtick
+xticks = [-0.8, 28, 58, 89.1, 120, 151, 182.9, -36, -63, -89, -117, -145, -151]
+yticks = [-3] + [-2] + [-1] + [-1] * 2 + [-1] + [-3] + [-7] + [-7] * 3 + [-5]
+
+for xtick, ytick, label in zip(xticks, yticks, labels):
+    ax.text(xtick, ytick, label, transform=ccrs.Geodetic(), fontsize=10)
+
+# Get slice of data at the 0th timestep
+p = ds.HGT.isel(time=0)
+
+# Use geocat-viz utility function to handle the no-shown-data
+# artifact of 0 and 360-degree longitudes
+slon = gvutil.xr_add_cyclic_longitudes(p, "lon")
+
+# Plot contour data at pressure level 5500 at the first timestep
+p = slon.plot.contour(ax=ax,
+                      transform=ccrs.PlateCarree(),
+                      linewidths=1.5,
+                      levels=[5500],
+                      colors='k',
```

Comment:
For readability use the full color name instead of the abreviation.
```suggestion
                      colors='black',
```